scripts/setup_kind_local_registry.sh

```diff
@@ -1,31 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Sine we delete this file we have to reference the new script in out CI setup: https://github.com/FoundationDB/fdb-kubernetes-operator/actions/runs/4357548931/jobs/7616992961. The files are located under `.github` in this repo.